source/profiles.txt

```diff
@@ -8,7 +8,8 @@ coastal/bathymetric lidar community, or the powerline mapping community). So as
 to not alter the LAS base specification, new classes can only be added to Point
 Data Record Formats 6-10, and classification values cannot start below 39. New
 attributes must be incorporated using Extra Bytes VLRs. It is strongly
-recommended that the development of Domain Profiles be coordinated, so as to
+recommended that the development of Domain Profiles be coordinated with the
+LAS committee on GitHub at https://github.com/ASPRSorg/LAS, so as to
```

Comment:
Should we link to this location or should we link to the main asprs.org website?

Reply:
Personally, I like the idea of guiding developers toward the GitHub page to avoid the overhead of ASPRS review. I believe LDPs are intended to be more fluid than the LAS spec itself.